homeassistant/components/nuki/binary_sensor.py

```diff
@@ -36,11 +36,6 @@ class NukiDoorsensorEntity(NukiEntity, BinarySensorEntity):
 
     _attr_device_class = BinarySensorDeviceClass.DOOR
 
-    @property
-    def name(self):
```

Comment:
We didn't implement `self._attr_name` as far as I can see.

https://github.com/home-assistant/core/pull/79806#discussion_r1087751212